src/python/zquantum/core/circuit/_gate.py

```diff
@@ -23,18 +23,18 @@
 
 class Gate(object):
     """Class for storing information associated with a quantum gate.
- 
+
     Attributes:
         name: string
             Name of the gate. Examples include 'H', 'CNOT' etc.
             The validity of the name goes hand in hand with the specific package
             for quantum computation, as described in 'label'.
         qubits: list[Qubit]
-            List of core.qubit.Qubit object that the gate operates on. 
+            List of core.qubit.Qubit object that the gate operates on.
         params: list[any]
-            List of parameters associated with the gate. 
+            List of parameters associated with the gate.
         The above three attributes follow a standard convention that is enforced throughout
-        Zap OS. 
+        Zap OS.
```

Comment:
I think we can drop "Zap OS" naming from here ;)

Reply:
bc7736d